app/assets/javascripts/form_exit.js.erb

```diff
@@ -1,9 +1,14 @@
 ;(function (w, d) {
   exitHandler = function () {
     var links = d.querySelectorAll('A');
+    var inSteps = window.location.href.indexOf('/steps/') >= 0;
+    if(!inSteps) return;
```

Comment:
Add a guard clause here that bails out if we're not ***within*** the steps pages. If we're not inside of those form pages, we really don't care where we click out to.